site/_posts/2021-02-16-orbitvu-fusionauth-story.md

```diff
@@ -0,0 +1,88 @@
+
+**Dan:** Why did you choose FusionAuth over the competition?
+
+**Maciej:** We've reviewed multiple solutions like Auth0, Okta, AWS Cognito, and MiniOrange but we decided to use FusionAuth due to its flexibility, great documentation, and competitive pricing. I'd say that flexibility (also understood as customizability) of the solution is where FusionAuth stands out. It was important to us that we can use FusionAuth both on-premise (including docker and Kubernetes) as well as a hosted, hassle-free Saas version. Thanks to the docker images and FusionAuth's Kickstart system we could easily integrate FusionAuth into our development process to build reproducible development environments.
+
+Another thing about flexibility, where the competition is far behind, is the possibility to customize the look and feel of almost every aspect of the FusionAuth using themes. By the way, personally I don't like FreeMarker's syntax but it does the job! It was critical to us to be able to have our own layout applied to login pages and emails.
+
+One more important factor that we considered was the way to migrate from our existing solution to the new one. Again, FusionAuth's way of doing this fitted our needs perfectly and made it possible to migrate user password hashes from the Django-based system to FusionAuth flawlessly (initially we've created our own password encryptor to handle this but turned out the built ones were enough).
+
+Last but not least is that at Orbitvu we love open source and we use it a lot. What we do like in FusionAuth is that, while not open source, it feels like an open source project due to its development style with Github's issue tracker.
+
+> We’re integrating with FusionAuth using its API and it works flawlessly.
+
+**Dan:** I see the Orbitvu website supports multiple languages; was the ability to localize FusionAuth important as well?
+
+**Maciej:** Definitely, it was one of the must-have features when we were looking for our auth solution.
+
+**Dan:** How much time and money would you say FusionAuth has saved you?
+
+**Maciej:** Given that FusionAuth is the only solution that is so complete and can be used on-premise without extra costs I'm pretty sure it has saved a lot of money for us. We use FusionAuth on our own hosting and we're managing it on our own so the costs are the hosting cost plus our working hours' cost. After the initial setup, there is not much to do with FusionAuth except upgrading from time to time, so this way it is way more cost-effective than the competing solutions that are sometimes extremely expensive when it comes to a bigger number of users.
```

Comment:
Given that FusionAuth is a complete solution that can be used on-premise without extra costs, I'm pretty sure it has saved us a lot of money.